foundation/donate.dd

```diff
@@ -35,28 +35,44 @@ $(DONATE_ITEM Donate through Flipcause, users,
     Flipcause helps non-profit organizations like the D Language Foundation manage their
     donations and provide more value to donors. Donating through Flipcause can enable us
     to more efficiently put your donation to use. You can choose to donate to our General
-    Fund or to a specific campaign. The menu below displays the campaigns currently available.
-    Click on 'Make a Donation' under any campaign to see more details for that campaign.
+    Fund or to a specific campaign. Click on one of the images below to visit a campaign
+    site for the details.
+    $(BR)$(BR)
+    If you have trouble donating through Flipcause or would simply prefer another supported
+    payment processor, you can still contribute to one of the campaigns through the other
+    options on this page. Please be sure to leave a note with the donation telling us which
+    campaign you'd like the funds to go to!
     $(BR)$(BR)
 
-    <iframe style="height:615px; width:95%; border:1px solid #dddddd; " src="https://www.flipcause.com/widget/NDMzMzI=" scrolling="no"></iframe>
+    $(TABLE 
+        $(TR 
+        $(TD <a href="https://www.flipcause.com/secure/cause_pdetails/NDkzNjc=">
+                <img src="$(ROOT_DIR)/images/campaign-forums.png"></a>)
```

Comment:
No slash after root dir.
Yup, it's pretty confusing...

Reply:
Crap. It displays fine on my local copy. I'll change it, though.